Workflow/CIDER_Workflow.ipynb

```diff
@@ -363,57 +364,37 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "ccac373a",
    "metadata": {},
    "source": [
     "## Testing for duplicates  \n",
-    "Before further comparison between the datasets, it might be important to know if or rather how many duplicates are in a single dataset. There is also the option to get the identifier for those duplicates with the parameter 'identifier=True'.  \n",
+    "Before further comparison between the datasets, it might be important to know if or rather how many and which duplicates in a single dataset. The duplicates are noted through their string identifier and their index number in the MolSupplier Object.\n",
     "(Short reminder: This method cannot be used unless the 'get_identifier_list_key' method is run previously!)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 15,
    "id": "136cef24",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2023-02-08 13:17:46,426 [INFO] CIDER: Number of duplicates in set_chlorbenzene-5.sdf: 0\n",
-      "2023-02-08 13:17:46,431 [INFO] CIDER: Number of duplicates in set_chlorbenzene.sdf: 0\n",
-      "2023-02-08 13:17:46,438 [INFO] CIDER: Number of duplicates in set_phenole.sdf: 2\n",
-      "2023-02-08 13:17:46,443 [INFO] CIDER: Updated dictionary with 'number_of_duplicates'\n"
+      "2023-06-02 18:10:11,421 [INFO] CIDER: Number of duplicates in set_chlorbenzene-5.sdf: 0, duplicate identifier(s): [], duplicate index: []\n",
+      "2023-06-02 18:10:11,423 [INFO] CIDER: Number of duplicates in set_chlorbenzene.sdf: 0, duplicate identifier(s): [], duplicate index: []\n",
+      "2023-06-02 18:10:11,425 [INFO] CIDER: Number of duplicates in set_phenole.sdf: 2, duplicate identifier(s): ['O=Cc1ccc(O)cc1', 'O=Cc1ccccc1O'], duplicate index: [44, 47]\n",
```

Comment:
There are only the indices of one of the two detected duplicates reported here. I think you simply have to execute the notebook again, it works as expected for me.

Reply:
Yes, I did the same and changed it.